app/components/avo/actions_component.html.erb

```diff
@@ -32,20 +32,8 @@
   >
     <div data-target="actions-list" class="w-full space divide-y">
       <% actions.each_with_index do |action, index| %>
-        <%= link_to action_path(action),
-          data: {
-            action_name: action.action_name,
-            'turbo-frame': Avo::ACTIONS_TURBO_FRAME_ID,
-            'action': 'click->actions-picker#visitAction',
-            'actions-picker-target': action.standalone ? 'standaloneAction' : 'resourceAction',
-            'disabled': is_disabled?(action),
-            # for some reason InstantClick fetches the URL even if it's prefetched.
-            turbo_prefetch: false,
-          },
-          title: action.action_name,
-          class: "flex items-center px-4 py-3 w-full font-semibold text-sm hover:bg-primary-100 #{is_disabled?(action) ? 'text-gray-500' : 'text-black'}" do %>
-          <%= svg 'play', class: 'h-5 mr-1 inline pointer-events-none' %> <%= action.action_name %>
-        <% end %>
+        <% next_action = actions[index + 1] %>
+        <%= render_item(action, next_action)%>
```

Comment:
```suggestion
        <%= render_item(action)%>
```

Reply:
Yes, we don't need next_action any more.